cmd/cosign/cli/sign/sign.go

```diff
@@ -130,7 +130,7 @@ func SignCmd(ctx context.Context, ko KeyOpts, regOpts options.RegistryOptions, a
 		}
 	}
 
-	// Set up an ErrDone considerion to return along "success" paths
+	// Set up an ErrDone consideration to return along "success" paths
```

Comment:
This puzzled me for a long time. I eventually gave up and just accepted the tool's suggestion.

A project person should suggest a reworded sentence.

Reply:
@dlorenc: I don't suppose you could suggest an alternative for this line?

Reply:
Haha whoa, no idea...

Maybe just `set up an ErrDone variable to return along "success" paths`?